client-python/grakn/service/Session/Concept/ConceptHierarchy.py

```diff
@@ -0,0 +1,579 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from typing import Union, Optional
+
+from grakn.service.Session.util import enums
+from grakn.service.Session.util.RequestBuilder import RequestBuilder
+# from grakn.service.Session.util.ResponseConverter import ResponseConverter # cannot get name from module because circular imports
+# import grakn.service.Session.util.ResponseConverter as ResponseConverter # toplevel only allowed
+from grakn.service.Session.util import ResponseConverter
+from grakn.service.Session.Concept import ConceptFactory
+from grakn.exception.ClientError import ClientError 
+
+
+
+class Concept(object):
```

Comment:
Why is this class not just called `Concept.py` since all the nested classes are effectively just further extensions of the `Concept` class? @flyingsilverfin @lolski

Reply:
Yeah can be renamed to Concept.py